tutor/specs/components/task-plan/homework/__snapshots__/add-exercises.spec.cjsx.snap

```diff
@@ -81,8 +81,18 @@ exports[`choose exercises component matches snapshot 1`] = `
             />
           </div>
           <span>
-            Tutor Selections
+            OpenStax Tutor Selections
           </span>
+          <button
+            aria-describedby="homework-selections-popover"
+            className="btn btn-link"
+            disabled={false}
+            id="homework-selections-trigger"
+            onClick={[Function]}
+            type="button"
+          >
+            What are these?
+          </button>
         </div>
         <div
```

Comment:
Also not a big deal but the Tour mask isn't including the new "what are these"

![screen shot 2017-06-16 at 9 28 57 am](https://user-images.githubusercontent.com/79566/27230838-3cebd780-5276-11e7-9c2c-76a2973e5c23.png)